db/migrate/20240227070217_create_bookings.rb

```diff
@@ -0,0 +1,14 @@
+class CreateBookings < ActiveRecord::Migration[7.1]
+  def change
+    create_table :bookings do |t|
+      t.date :start_date
+      t.date :end_date
+      t.integer :no_of_guests
+      t.boolean :host_approve
```

Comment:
```suggestion
      t.boolean :host_approve
```
      t.boolean :host_approved